### PR TITLE
Update RTD. Use sphinx-build-2

### PR DESCRIPTION
**Pull request by phracek**
Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

This Pull Request fixes issues catched during document generation by sphinx.

### CODE REVIEW

File: moduleframework/dockerlinter.py [hunk 1]
```diff
@@ -184,15 +184,18 @@ def check_from_directive_is_valid(self):
 
     def check_chained_run_dnf_commands(self):
         """
-        Function checks if Dockerfile does not contain more `RUN dnf/yum` commands
-        in more then one row.
+        Function checks if Dockerfile does not contain more `RUN dnf/yum` commands in more then one row.
```
Comment (bkabrda) [inline moduleframework/dockerlinter.py]: The docstring might make more sense as `This function checks that there are no consecutive RUN commands executing dnf/yum in the Dockerfile, as these need to be chained.`

File: Makefile.docs [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 # You can set these variables from the command line.
 SPHINXOPTS    =
-SPHINXBUILD   = sphinx-build
+SPHINXBUILD   = sphinx-build-2
```
Comment (bkabrda) [inline Makefile.docs]: Why is this necessary? Why doesn't just `sphinx-build` work?

Comment (phracek) [reply Makefile.docs]: It runs sphinx-build in python-3 environment.

Comment (phracek) [reply Makefile.docs]: and avocado does not have python-3 packages https://github.com/avocado-framework/avocado/issues/2024

**Comment by bkabrda** [comment]
I'm not merging myself, since review from @jscotka has been requested and I'll leave this to him to review and merge.
